bundles/org.eclipse.osgi.services/META-INF/MANIFEST.MF

```diff
@@ -2,56 +2,38 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %osgiServices
 Bundle-SymbolicName: org.eclipse.osgi.services
-Bundle-Version: 3.10.200.qualifier
+Bundle-Version: 3.11.0.qualifier
 Bundle-Description: %osgiServicesDes
 Bundle-Localization: plugin
 Bundle-Vendor: %eclipse.org
 Bundle-DocUrl: http://www.eclipse.org
 Bundle-ContactAddress: www.eclipse.org
-Export-Package: org.osgi.service.cm;version="1.6";uses:="org.osgi.framework",
- org.osgi.service.component;version="1.4";uses:="org.osgi.framework",
- org.osgi.service.component.annotations;version="1.3",
- org.osgi.service.component.runtime;version="1.4";uses:="org.osgi.framework,org.osgi.util.promise,org.osgi.service.component.runtime.dto",
- org.osgi.service.component.runtime.dto;version="1.4";uses:="org.osgi.dto,org.osgi.framework.dto",
- org.osgi.service.device;version="1.1";uses:="org.osgi.framework",
- org.osgi.service.event;version="1.4";uses:="org.osgi.framework",
+Export-Package: org.osgi.service.component.annotations;version="1.3",
  org.osgi.service.http;version="1.2.1";uses:="javax.servlet,javax.servlet.http",
  org.osgi.service.http.context;version="1.1";uses:="org.osgi.framework,javax.servlet.http",
  org.osgi.service.http.runtime;version="1.1";uses:="org.osgi.service.http.runtime.dto",
  org.osgi.service.http.runtime.dto;version="1.1";uses:="org.osgi.dto,org.osgi.framework.dto",
  org.osgi.service.http.whiteboard;version="1.1",
- org.osgi.service.log;version="1.5";uses:="org.osgi.framework",
- org.osgi.service.metatype;version="1.4";uses:="org.osgi.framework",
- org.osgi.service.provisioning;version="1.2",
- org.osgi.service.upnp;version="1.2",
- org.osgi.service.useradmin;version="1.1";uses:="org.osgi.framework",
- org.osgi.service.wireadmin;version="1.0.1";uses:="org.osgi.framework"
+ org.osgi.service.log;version="1.5";uses:="org.osgi.framework"
```

Comment:
We could get rid of the actual classes from the `org.osgi.service.log` package by placing a mandatory 
matching attribute on the package.

```
  org.osgi.service.log;version="1.5";uses:="org.osgi.framework";mandatory:="never.ever.use";never.ever.use="true"
```

If I recall correctly, at least for the Equinox framework implementation, we consider the Import-Package of a package before allowing its export to be selected.  So if a bundle has an Import-Package which can never match its own export then its export will always be dropped.  Here the import would never be satisfied by our own export because of the mandatory directive for "never.ever.use".  This implies that the empty content from `org.eclipse.osgi.services` for the `org.osgi.service.log` package could never be used because it would always be substituted by the import to something real.

We would have to validate that is true.

Reply:
That's an interesting suggestion. I will try and validate that tomorrow.
It would be great if we can delete the service.log package too.

Reply:
> This implies that the empty content from `org.eclipse.osgi.services` for the `org.osgi.service.log` package could never be used because it would always be substituted by the import to something real.
> 
> We would have to validate that is true.

I just tested it with a test project that contained `Require-Bundle: org.eclipse.osgi.services` in its manifest and and uses the Logger interface from `org.osgi.service.log`. The suggested approach worked well with PDE, when build with Tycho and when launched into an Eclipse application. The classes from `org.osgi.service.log` were available to the compiler respectively could be loaded at runtime. At least at compile-time the `mandatory` was not necessary. But I think it is better to be save.

I removed the embedded sources from `org.osgi.service.log` but added a not to not have a empty directory to silence PDE.